components/status/StatusCard.vue

```diff
@@ -171,7 +173,7 @@ const forceShow = ref(false)
                 </div>
               </div>
             </div>
-            <StatusActionsMore v-if="actions !== false" :status="status" me--2 />
+            <StatusActionsMore v-if="actions !== false" :status="status" me--2 @after-delete="props.deleteItem" />
```

Comment:
I'm a fan now of using `emit`. I got convinced that this syntax makes it more clear that this child outputs a signal to its parent. Open to what others thing.